sdk/spring/azure-identity-spring/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.6.0 (2021-06-23)
-
+### New Features
```

Comment:
This is not a feature, how about change it to `### Dependency Upgrades`

Refs: https://github.com/spring-projects/spring-boot/releases

Reply:
And this PR has 18 files changed, I think we will release more than 18 artifacts. Did you need to change more files?

Reply:
I checked it one by one, I changed only files with empty entry